src/main/java/mServer/crawler/sender/br/WebAccessHelper.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * WebAccessHelper.java
+ * 
+ * Projekt    : MLib
+ * erstellt am: 04.10.2017
+ * Autor      : Sascha
+ * 
+ */
+// Ist eigentlich aus MLib in der neuen Architektur, aber für den neuen BR-Crawler erstmal hierher kopiert
+package mServer.crawler.sender.br;
+
+import de.mediathekview.mlib.Const;
+import java.net.URL;
+
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+import mServer.crawler.FilmeSuchen;
+import mServer.crawler.RunSender;
+
+public class WebAccessHelper {
+    
+    public static String getJsonResultFromPostAccess(URL serverUrl, String request) throws IllegalArgumentException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.IllegalArgumentException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)